iscsiuio/src/unix/meson.build

```diff
@@ -0,0 +1,37 @@
+#
+# meson build file for iscsiuio/src/unix
+#
+
+# build date files since 'libs' needs them
+
+build_date_cmd = find_program('build_date.sh')
+
+src_date_epoch = get_option('source_date_epoch')
+epoch_date_arg = []
+message('Source Date Epoch: ' + src_date_epoch)
+if src_date_epoch != 'NONE'
+  epoch_date_arg = ['-s', src_date_epoch]
+endif
```

Comment:
Yes, another good catch. That "option" was never listed in meson_options.txt anyway!

I put this in just in case I couldn't get the environment passing to work, but it works, so no need for it.